WooCommerce/Classes/POS/Presentation/Settings/PointOfSaleSettingsHardwareDetailView.swift

```diff
@@ -20,19 +25,242 @@ struct PointOfSaleSettingsHardwareDetailView: View {
+        case .setup:
+            showBarcodeScanningSetupModal = true
+        case .documentation:
+            showBarcodeScanningDocumentationModal = true
+        }
+    }
+
+    private var cardReadersView: some View {
+        VStack(spacing: POSSpacing.large) {
+            VStack(spacing: POSSpacing.medium) {
+                VStack(spacing: POSPadding.small) {
+                    Text(Localization.cardReadersDescription)
                         .font(.posBodyLargeRegular())
-                    Text("\(destination.title) settings")
+                        .multilineTextAlignment(.center)
+                    Text(Localization.cardReadersSubtitle1)
+                        .font(.posBodyMediumRegular())
+                        .foregroundStyle(.secondary)
+                        .multilineTextAlignment(.center)
+                    Text(Localization.cardReadersSubtitle2)
+                        .font(.posBodyMediumRegular())
+                        .foregroundStyle(.secondary)
+                        .multilineTextAlignment(.center)
+                    Text(Localization.cardReadersSubtitle3)
                         .font(.posBodyMediumRegular())
-                    Text("Some placeholder")
+                        .foregroundStyle(.secondary)
+                        .multilineTextAlignment(.center)
+                }
+            }
+            .padding()
+
+            if let lastKnownLoadedCardReader {
+                HStack {
+                    Text("Model: \(lastKnownLoadedCardReader)")
                         .font(.posBodyMediumRegular())
                         .foregroundStyle(.secondary)
+                        .multilineTextAlignment(.center)
                 }
-                .padding()
-                .navigationTitle(destination.title)
             }
+
+            List {
+                Button {
+                    showCardReaderDocumentationModal = true
+                } label: {
+                    HStack(alignment: .firstTextBaseline) {
+                        Image(systemName: "doc.text")
+                            .font(.posBodyLargeRegular())
+                        VStack(alignment: .leading, spacing: POSPadding.xSmall) {
+                            Text(Localization.cardReaderDocumentationTitle)
+                                .font(.posBodyLargeRegular())
+                            Text(Localization.cardReaderDocumentationSubtitle)
+                                .font(.posBodyMediumRegular())
+                                .foregroundStyle(.secondary)
+                        }
+                    }
+                }
+                .buttonStyle(.plain)
+            }
+        }
+        .navigationTitle(Localization.cardReadersTitle)
+        .posFullScreenCover(isPresented: $showCardReaderDocumentationModal) {
+            SafariView(url: WooConstants.URLs.inPersonPaymentsLearnMoreWCPay.asURL())
         }
+        .task { @MainActor in
+            // TODO: WOOMOB-1172
```

Comment:
This is temporary, most likely we'll fetch card reader information from passing in either a reference to the aggregate model or expose its card reader details.

Reply:
Jinx! I just had the same question when reviewing your PR :D 

I also think that in this case, for settings, it makes more sense to go through the aggregate, as we'll deal with settings across different features that are part of such aggregate, but we'll see what fits better 👍